src/overlays/actors/ovl_Obj_Warpstone/z_obj_warpstone.h

```diff
@@ -5,14 +5,30 @@
 
 struct ObjWarpstone;
 
-typedef void (*ObjWarpstoneActionFunc)(struct ObjWarpstone* this, GlobalContext* globalCtx);
+typedef s32 (*ObjWarpstoneActionFunc)(struct ObjWarpstone* this, GlobalContext* globalCtx);
+
+typedef enum {
+    /* 0 */ SEK_MODEL_CLOSED,
+    /* 1 */ SEK_MODEL_OPENED
+} ObjectSekModels;
```

Comment:
What does `SEK` mean? It isn't immediately obvious to me, it's always worth being cautious for over-abbreviation

Another comment is that I've more often seen `OPEN` and `CLOSE` instead of `OPENED` and `CLOSED`. I think I'd prefer the former but I'm not that picky and am fine if you leave it as is

Reply:
ObjectSek is the name of the owl statue's object file (at least when opened in Z64Utils). Since this particular enum is about the object file, I named it Sek instead of Warpstone. I'm imitating Anghelo's Romani/Cremia with respect to this.